docs/API.md

```diff
@@ -158,6 +158,11 @@ their equivalent) can be accessed as multiple sizes. The default size is
 variants, which can accessed by adding the `_small` or `_large` suffix
 to the avatar URL.
 
+If there is no avatar set by the user, then the server will provide a url to a default avatar image.
+Keep in mind that this default url will have an extension `.png`, unlike avatars that are set by users.
+You can access `_small` and `_large` versions by changing the file name to `default_small.png` and `default_large.png`
```

Comment:
@rfk thoughts on this madness?

Reply:
Is it necessary for us to serve it with a `.png` extension, or would we just serve it at some well-known id?  Since we're doing this to simplify the work of reliers, sending something like https://firefoxusercontent.com/00000000000000000000000000000000 might be more useful than sending something they have to special-case.  (This would apply doubly-so to constructing the `_small` and `_large` variants).